scrapers/MyDirtyHobby.yml

```diff
@@ -98,4 +99,11 @@ driver:
           Domain: ".mydirtyhobby.com"
           Value: "1"
           Path: "/"
-# Last Updated October 02, 2023
+  cookies:
```

Comment:
This line can be removed: all cookies should be defined in the same `cookies` map :)
```suggestion
```